### ISSUE TITLE
no member 'OnTaskComplete' when building for node

### ISSUE THREAD

**Issue by aaronik** [line/stellite#68]
Hey y'all, 

Thank you so much for building this! My company is looking for a QUIC server to use with node.js and finding stellite has been a huge boon!

Unfortunately I've been having a lot of trouble getting it to build. Here's an image of my troubles:

![image](https://user-images.githubusercontent.com/1324601/42841278-63b567f2-89be-11e8-8637-d79c1922893d.png)

For reference, the command I'm running:
`./tools/build.py  --chromium-path /third_party/chromium_linux/ --target-platform linux --target node_stellite --target-type node_module --node-module-version 46 build`

The error:
`../../node_binder/node_http_fetcher.cc:87:10: error: no member named 'OnTaskComplete' in 'stellite::HttpFetcher'`

Note this happens with our without the `--node-module-version` flag.

I've run into this same issue from ubuntu 14.04 and ubuntu 16.04 docker containers, although I can't see that making much of a difference.

Hopefully somebody else has run into this issue too. Thanks in advance for all your help!

**Comment by ptyjj** [comment]
@Aaronik 

Hello. May I see your build enviromnent? This quic node server builded very limited build environment. (For example, macos hi sierra sdk can't build this project).

Thank you.

**Comment by Aaronik** [comment]
Absolutely, I can give you a very specific build environment. And thank you for your quick response! I really appreciate your help.

The machine is a docker ubuntu 16.04 image: `ubuntu:xenial`. Specifically, the command to boot that is:

```sh
sudo docker run -it --rm ubuntu:xenial
```

Then I've installed the following libraries, which I've discovered by trial and error:

```sh
sudo apt-get update && sudo apt-get install -y libkrb5-dev libgconf2-dev pkg-config lsb-release curl python git build-essential libnss3-dev libfreetype6-dev
```

The host machine, although I don't _think_ it should matter, is Ubuntu 18.04.

```
Ξ uname -a
Linux ******** 4.15.0-24-generic #26-Ubuntu SMP Wed Jun 13 08:44:47 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by jtyang** [comment]
您好，邮件已经收到，我会尽快回复您！祝您开心...
